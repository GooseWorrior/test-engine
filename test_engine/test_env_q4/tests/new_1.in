newInit 1
[] 1 100
[]= 1 100 100
erase 1 100
size 1
