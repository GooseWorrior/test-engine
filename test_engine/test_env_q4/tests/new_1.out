1: (1,2) (3,4) (5,6) (7,8)
0
1: (1,2) (3,4) (5,6) (7,8) (100,0)
1: (1,2) (3,4) (5,6) (7,8) (100,100)
1: (1,2) (3,4) (5,6) (7,8)
4
1: (1,2) (3,4) (5,6) (7,8)
